### DISCUSSION TITLE
Get epic summary from key name?

### DISCUSSION THREAD

**Discussion by guysv** [ankitpokhrel/jira-cli#445] [Q&A] [answered]
I have my epic key, but `jira` only seems to support listing subitems of epic with "list"? there's no equivalent "view" command for epics.

I'd also like to use it in a script so if I somehow can just resolve the summary from the epic key, that would be great. (so no pretty-printing please, json is even better)

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @guysv, you can use `jira issue view <KEY>` to view epic descriptions. If you want to get a summary of an issue/epic with a key you can do something like this.

```sh
jira issue list -q"key=ISS-1" --plain --no-headers --columns=summary | awk '{$1="";print $0}'
```

**Reply by guysv** [reply]
thanks!
